src/views/Simulation/components/TopBar/MapEntities.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Checkbox,
+  MenuItem,
+  FormControl,
+  Select,
+  ListItemText,
+  InputLabel,
+  Chip,
+  Box,
+  Typography,
+  Switch,
+} from '@mui/material';
+import { useSimulationViewContext } from '../../SimulationViewContext';
+import { MAP_VIEW_FILTERS } from '../../constants/settings';
+
+export default function MapEntitiesDropdown() {
+  const { selectedMapFilters, setSelectedMapFilters, onlyCriticalPoints, setOnlyCriticalPoints } =
+    useSimulationViewContext();
+
+  const handleChange = (event) => {
+    const value = event.target.value;
+
+    if (value.includes('all')) {
+      setSelectedMapFilters(MAP_VIEW_FILTERS.map((e) => e.value));
+    } else {
+      setSelectedMapFilters(value);
+    }
+  };
```

Comment:
I think there might be a bug in the dropdown list (but I'm not sure if this would be in this function ^^). Once the "all" option has been checked, it is impossible to "uncheck" the other boxes in the dropdown list (clicking on the Chip to remove it works though)

Reply:
@arianber7 there's still a bug in the dropdown list: when the "all" option is selected, trying to uncheck one of the other boxes lead to all boxes being unchecked. The expected behavior would be to have only this box unchecked, while all other boxes should remain checked.
Also, when the "all" option is selected, it disables the "only critical points option", but I think these two options should be independent: the "all" option should only impact the checkboxes